pkg/client/active_directory_membership_object_set.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Hewlett Packard Enterprise Development LP
+
+package client
+
+import (
+	"reflect"
+
+	"github.com/hpe-storage/common-host-libs/jsonutil"
+	"github.hpe.com/nimble-dcs/golang-sdk/pkg/client/v1/model"
+	"github.hpe.com/nimble-dcs/golang-sdk/pkg/util"
+)
+
+
+// Manages the storage array&#39;s membership with the Active Directory.
```

Comment:
nit: Need to remove those unwanted characters in  `array&#39;s`
`Manages the storage array membership with the Active Directory`

Reply:
Sure, will fix it .